lepton/image.go

```diff
@@ -173,7 +173,7 @@ func addFilesFromPackage(packagepath string, m *fs.Manifest) {
 
 		err = m.AddFile(filePath[1], hostpath)
 		if err != nil {
-			return err
+			log.Error(err)
```

Comment:
I think this should be log.Error(err) && then return err otherwise it hits that return nil (which could prob just be removed)

Reply:
That would stop the walk of sysroot completely and prevent any further files from being added to the image. I think we could do that, but we should probably check addFilesFromPackage and return an error in BuildPackageManifest so the command would fail and make you fix the package. We have to return nil if we want the behavior to be what it was before, where it ignored the error.

Reply:
oh yeh, you're right